LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 Protacon
+Copyright (c) 2018 Pinja
```

Comment:
I would assume copyright year should also be updated, but dunno - not a big issue.